### PR TITLE
5.1.5 docs releasenotes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.1.md [hunk 1]
```diff
@@ -11,6 +11,65 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+
+### New Features
+
+- **I95-31265 Software Management Button on Router Page:** Users are now able to upgrade a single router from the Router Page using a button located at the top of the Router Page.
+------
+- **I95-35347 Add hostname to GUI DHCP lease table:** A Hostname column has been added to the GUI in the DHCP Server Table for DHCP Leases.
+------
+- **I95-39236/I95-40804/I95-41531 Session capture to produce unique pcap files:** A capture tag has been added to the session-capture feature. If a session-capture is created with a tag, that tag is used to name the pcaps throughout the topology, instead of the service name. Users are now able to initiate this capture from the Services page in the GUI and the command line.
+------
+- **I95-39617 Add cost (currency) and carrier information to the `network-interface` and `authority`:** Added attributes to the network-interface that reflect the weighted cost for paths. For more information, refer to [`network-interface`](config_reference_guide.md/#network-interface), and [`authority`](config_reference_guide.md/#authority) in the Element Reference Guide.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:25]: Replace "the weighted" with "the monetary". We do not want to confuse the dollar value as a influencer of path selection. This feature was added as a means of translating the SVR savings tool into monetary savings.

File: docs/release_notes_128t_5.1.md [hunk 2]
```diff
@@ -11,6 +11,65 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-40335 Support running KVM on the host:** Addressed deprecated-packages dependency conflicts with kvm, and updated the minimum allowed version of libX11 to address CVE-2020-14363.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:27]: The details of why are not pertinent. Recommended rewording:
"KVM packages and corresponding dependencies have been added to the SSR distribution"

File: docs/release_notes_128t_5.1.md [hunk 3]
```diff
@@ -11,6 +11,65 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-40776 GUI self diagnostics for client connectivity:** A client connectivity error now diplays a dialog with several steps intended to help the user diagnose issues.
+
+### Resolved Issues
+
+- **I95-36099 Time range selection for analytics in the GUI is difficult to use:** Now using the native implementation for date-time.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:33]: Replace "Now using the native implementation for date-time."
With "Replaced with browser-native date-time selector."

File: docs/release_notes_128t_5.1.md [hunk 4]
```diff
@@ -11,6 +11,65 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-38731 Mars systemd service fails to start after too many restarts:** This issue has been resolved.
+------
+- **I95-40009 Not responding to ARP request from source 0.0.0.0. on a DHCP interface:** The SSR now responds to ARP requests from source 0.0.0.0.
+------
+- **I95-40016 Adaptive Encryption cause packet drop on HA fabric:** Resolved an `Adaptive encryption out of order` modification error causing packet drop on HA fabric.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:39]: Replace "Adaptive Encryption cause packet drop on HA fabric"
with "Flow-migration with adapative-encryption may cause packet drops"

File: docs/release_notes_128t_5.1.md [hunk 5]
```diff
@@ -11,6 +11,65 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-40035 Jute maxbuffer limit for running/candidate configurations:** Created an alarm when the zookeeper jute buffer exceeds a threshold (75%) and an alert to change the system environment config.
+------
+- **I95-40060 `show session captures` not displaying active captures:** Updated the query argument to correctly display session captures for all services.
+------
+- **I95-40624 Occasional failures in time-based HMAC:** This issue has been resolved.
+------
+- **I95-40894 Firewall pinhole test:** Added an acceptable loss of packets threshold for FirewallPinhole tests.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:47]: Replace "I95-40894 Firewall pinhole test:** Added an acceptable loss of packets threshold for FirewallPinhole tests."
with "I95-40894 Firewall pinhole false-positives:** Added an acceptable loss of packets threshold for Firewall Pinhole tests.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.1.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

File: docs/release_notes_128t_5.1.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

File: docs/release_notes_128t_5.1.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

File: docs/release_notes_128t_5.1.md (hunk 4 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

File: docs/release_notes_128t_5.1.md (hunk 5 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

**Review by MichaelBaj — DISMISSED** [review]

**Review by jeffmart — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart — APPROVED** [review]

**Pull request merged**